frontend/src/components/Header.jsx

```diff
@@ -1,6 +1,7 @@
+import PropTypes from "prop-types";
 import SearchBar from "./SearchBar";
 
-function Header() {
+function Header({ userSearch, onSubmit, onChange }) {
   return (
     <div className="header">
```

Comment:
This div seems to be a header, use correct html tag.

Reply:
Ok, it'd done.